lib/product-import.coffee

```diff
@@ -212,17 +219,25 @@ class ProductImport
     skuString = "sku in (#{skus.map((val) -> JSON.stringify(val))})"
     "masterVariant(#{skuString}) or variants(#{skuString})"
 
+  _doesProductHaveSku: (product) ->
+    if product.masterVariant?.sku
+      return true
+
+    if product.variants?.length
+      for variant in product.variants
+        if variant.sku
+          return true
+    false
```

Comment:
> return false?

Coffeescript returns the last value as a function result.
I will check if it works - I was focusing only the code where we check the existence of a product on the API.